timestream.lua

```diff
@@ -0,0 +1,339 @@
+-- speeds up the calendar, units, or both
+
+--[====[
+
+timestream
+==========
+This is only for fortress mode.
+
+Usage:
+:timestream <scalar> <fps> <simulate units y/n>
+Examples:
+
+:timestream     2:          Calendar runs at x2 normal speed
+:timestream    -1 100:      Calendar runs at dynamic speed to simulate 100 FPS
+:timestream    -1 100 1:    Calendar & units are simulated at 100 FPS
+
```

Comment:
Might be nice to add a link to https://gist.github.com/IndigoFenix/cf358b8c994caa0f93d5 around here

Reply:
Done